atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransactionManager.java

```diff
@@ -106,6 +106,8 @@ public static SerializableTransactionManager create(KeyValueService keyValueServ
             int defaultGetRangesConcurrency,
             boolean initializeAsync,
             Supplier<Long> timestampCacheSize) {
+        TimestampTracker timestampTracker = TimestampTrackerImpl.createWithDefaultTrackers(
+                timelockService, cleaner, initializeAsync);
```

Comment:
It's rather odd to have dependent objects (the `TimestampTracker` mixed up with random config parameters (starting from `allowHiddenTableAccess`). But the fix for that is probably beyond the scope of this PR (and breaking) :(

Reply:
But if we haven't released TimestampTracker, we can put it where we like in the constructor! :tada: